### PR TITLE
title when sharing note

### PR DESCRIPTION
**Pull request by gonzalovelasco**

### CODE REVIEW

File: lib/screens/note/components/note_editor_app_bar.dart [hunk 1]
```diff
@@ -40,9 +40,12 @@ class NoteEditorAppBar extends ConsumerWidget {
       return;
     }
     final box = context.findRenderObject() as RenderBox?;
+    String? title = titleController?.text;
+    String concatenatedText = title != null ? "$title\n$content" : content;
```
Comment (matthewwong525) [inline lib/screens/note/components/note_editor_app_bar.dart:44]: good work on this!

File: lib/screens/note/components/note_editor_app_bar.dart [hunk 2]
```diff
@@ -29,9 +29,9 @@ class NoteEditorAppBar extends ConsumerWidget {
   final TextEditingController? titleController;
 
   void _onShare(BuildContext context) async {
+    String content = contentController?.text ?? "";
     if (kIsWeb) {
-      await Clipboard.setData(
-          ClipboardData(text: contentController?.text ?? ''));
+      await Clipboard.setData(ClipboardData(text: content));
```
Comment (matthewwong525) [inline lib/screens/note/components/note_editor_app_bar.dart:34]: this should include the title as well `$title\n$content`

**Review by matthewwong525 — CHANGES_REQUESTED** [review]
When sharing from web, it should include the title as well!

File: lib/screens/note/components/note_editor_app_bar.dart (hunk 2 above)
Comment (gonzalovelasco) [reply lib/screens/note/components/note_editor_app_bar.dart]: Fixed 502ad25f4d55a6edd7ce568f81226b93b559e5ae

**Review by matthewwong525 — APPROVED** [review]
LGTM

**Pull request merged**